src/main/java/xyz/nucleoid/mineout/game/MineoutActive.java

```diff
@@ -286,22 +290,47 @@ private void tickClosing(long time) {
         }
     }
 
+    private static String ordinal(int i) {
+        String[] suffixes = new String[] { "th", "st", "nd", "rd", "th", "th", "th", "th", "th", "th" };
+        switch (i % 100) {
+            case 11:
+            case 12:
+            case 13:
+                return i + "th";
+            default:
+                return i + suffixes[i % 10];
+        }
+    }
+
+    @SuppressWarnings("OptionalGetWithoutIsPresent")
     private void broadcastFinish() {
-        Text message;
+        PlayerSet players = this.gameSpace.getPlayers();
         if (!this.finishRecords.isEmpty()) {
             FinishRecord winningRecord = this.finishRecords.get(0);
 
-            message = new LiteralText("")
-                    .append(new LiteralText(winningRecord.player.getName()).formatted(Formatting.AQUA))
-                    .append(" won the game!")
-                    .formatted(Formatting.GOLD);
+            Text message = new LiteralText("").append(winningRecord.player.getName()).append(" was 1st!").formatted(Formatting.GOLD);
+            players.forEach(player -> {
+                if(players.stream().count() == 1) return;
+                Text subtitle;
+                if(this.finishRecords.stream().anyMatch(finishRecord -> finishRecord.player.getId().equals(player.getUuid()))) {
+                    FinishRecord record = this.finishRecords.stream().filter(rec -> rec.player.getId().equals(player.getUuid())).findFirst().get();
```

Comment:
Could rather use `Iterables.indexOf(this.finishRecords, record -> ...)` here and merge it with the above if statement for -1 check